spyder/plugins/variableexplorer/widgets/main_widget.py

```diff
@@ -571,3 +613,11 @@ def _set_actions_and_menus(self, nsb):
         # several places in CollectionsEditor.
         editor.insert_action_above = QAction()
         editor.insert_action_below = QAction()
+
+    def _enable_filter_actions(self, value):
+        """Handle the change of the filter state."""
+        self.exclude_private_action.setEnabled(value)
+        self.exclude_uppercase_action.setEnabled(value)
+        self.exclude_capitalized_action.setEnabled(value)
+        self.exclude_unsupported_action.setEnabled(value)
+        self.exclude_callables_and_modules_action.setEnabled(value)
```

Comment:
```suggestion
        self.exclude_callables_and_modules_action.setEnabled(value)

```

Add empty line at the end.